experiment/manual-trigger/manual-trigger.go

```diff
@@ -100,39 +102,45 @@ func sanityCheckFlags(o options) error {
 	return nil
 }
 
-func loadToken(path string) (string, error) {
-	raw, err := ioutil.ReadFile(path)
-	if err != nil {
-		return "", err
-	}
-	return strings.TrimSpace(string(raw)), nil
-}
-
 func main() {
 	o := flagOptions()
 	err := sanityCheckFlags(o)
 	if err != nil {
 		log.Fatal(err)
 	}
 
+	var tokens []string
+	tokens = append(tokens, o.githubTokenFile)
+
+	if o.jenkinsTokenFile != "" {
+		tokens = append(tokens, o.jenkinsTokenFile)
+	}
+
+	if o.jenkinsBearerTokenFile != "" {
+		tokens = append(tokens, o.jenkinsBearerTokenFile)
+	}
+
+	secretAgent := config.NewSecretAgent()
+	if err := secretAgent.Start(tokens); err != nil {
+		logrus.WithError(err).Fatal("Error starting secrets agent.")
+	}
+
+	getSecret := func(secretAgent *config.SecretAgent, secretPath string) func() []byte {
```

Comment:
nit: None of the `getSecret` functions require the `secretAgent *config.SecretAgent` parameter. `secretAgent` is already captured by the anonymous function closure.